### DISCUSSION TITLE
This emulator is still developed? WIFI Fix?

### DISCUSSION THREAD

**Discussion by Swappo91** [rafaelvcaetano/melonDS-android#909] [Q&A]
Hi rafael! I really love your work! I have just a question. Are continuing to develop this melon ds? Now the original emu for pc is on 0.9.4

And can you fix the wifi wfc problem?

What is the difficulty?

Thanks

**Comment by Fewnity** [comment]
I want the wifi support too, I'm going to release my Nintendo DS game with a multiplayer part :P

**Comment by Swappo91** [comment]
The developer dont reply since feb 2022 i think. Maybe this project actually is frozen?

**Comment by rafaelvcaetano** [comment]
Hey guys! Sorry for the delay. The project has not been abandoned but development has slowed down considerably because I have been very busy with other life stuff. I hope to resume development soon.

Regarding Wi-Fi, I don't have any news to share. For now, you should keep trying to use the workarounds suggested by other users.

**Reply by Swappo91** [reply]
Hi rafael! Thanks for your reply! Workaround not work properly unfortunally :( I really hope you can find a easy fix for that. Idk whats the problem but i think its fixable. Keep pushing and stay safe bro.

**Comment by ghost** [comment]
You can launch MelonDS via Termux by using the linux version.
Then you could get online support.

**Reply by ghost** [reply]
lack of opengl renderer
it will run very slow
the controls too.
